content/en/apps/guides/hosting/4.x/data-migration.md

```diff
@@ -0,0 +1,270 @@
+---
+title: "Migration from CHT 3.x to CHT 4.x"
+linkTitle: "Data migration to 4.x"
+weight: 10
+description: >
+  Guide to migrate existent data from CHT 3.x to CHT 4.x
+aliases:
+relatedContent: >
+---
+
+The hosting architecture differs entirely between CHT-Core 3.x and CHT-Core 4.x. Migrating data from an existing instance running CHT 3.x requires a few manual steps. 
+This guide will present the required steps while using a migration helping tool, called `couchdb-migration`. This tool interfaces with CouchDb, to update shard maps and database metadata.
+By the end of this guide, your CHT-Core 3.x CouchDb will be down and CHT-Core 4.x ready to be used.
+Using this tool is not required, and the same result can be achieved by calling CouchDb endpoints directly. [Consult CouchDB documentation for details about moving shards](https://docs.couchdb.org/en/3.2.2-docs/cluster/sharding.html#moving-a-shard). 
+
+### 1. Install CHT data migration tool
+
+Open your terminal and run these commands. They will create a new directory, download a docker compose file and download the required docker image. 
+```shell
+mkdir -p ~/couchdb-migration/ 
+cd ~/couchdb-migration/ 
+curl -s -o ./docker-compose.yml https://raw.githubusercontent.com/medic/couchdb-migration/main/docker-compose.yml
+docker-compose up
+```
+
+For the following steps, the tool needs access to your CouchDb installation. To allow this access, you will need to provide a URL to your CouchDB installation that includes authentication. 
+Additionally, if your CouchDb runs in docker, the tool needs to be added to the same docker network in order to access protected endpoints:
+
+```shell
+export COUCH_URL=http://<authentication>@<host-ip>:<port>
+```
+or 
+```shell
+export CHT_NETWORK=<docker-network-name>
+export COUCH_URL=http://<authentication>@<docker-service-name>:<port>
+```
+
+For simplicity, you could store these required values in an `.env` file: 
+```shell
+cat > ${HOME}/couchdb-migration/.env << EOF
+CHT_NETWORK=<docker-network-name>
+COUCH_URL=http://<authentication>@<docker-service-name>:<port>
+EOF
+```
```

Comment:
Per our chat on Slack today, these all need to be `https`, correct?  I think it's worth updating the URLs to be `https` and explicitly call out that the CouchDB Migration utility needs to talk over `https` when talking to Medic OS.